ACHNBrowserUI/ACHNBrowserUI/views/todayDashboard/TodayTasksSection.swift

```diff
@@ -11,23 +11,30 @@ import Backend
 
 struct TodayTasksSection: View {
     @EnvironmentObject private var collection: UserCollection
+    @Binding var sheet: Sheet.SheetType?
     
-    var isSunday: Bool {
-        Calendar.current.component(.weekday, from: Date()) == 1
+    private var tasksCount: Int {
+        collection.dailyCustomTasks.tasks.count
+    }
+    
+    private var rows: Int {
+        var rowsFloat = CGFloat(tasksCount) / 4.0
+        rowsFloat.round(.up)
+        return Int(rowsFloat)
```

Comment:
```suggestion
        let rowsFloat = CGFloat(tasksCount) / 4.0
        return Int(rowsFloat.rounded(.up))
```

Reply:
This is a point where I struggled extremely. Xcode told me it could not calculate value because it’s too nested. I think my MacBook is to old 😂 I’ll give it a new try.

Reply:
My Xcode wont build this...
```
Type '()' cannot conform to 'BinaryInteger'; only struct/enum/class types can conform to protocols
```

Reply:
Let me try on my branch, that's bizarre

Reply:
So that's weird, it's working on my computer but lower the build so much! So I trust you, just let the code like this then.